### PR TITLE
Added a more verbose quick start doc

### PR DESCRIPTION
**Pull request by lawrencegripper**
Currently work in progress but looking to explain in more detail the system.

### CODE REVIEW

File: docs/Quick Start.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+``` bash
+
+docker build -t lawrencegripper/ion-module-downloader -f ./modules/downloader/Dockerfile .
+docker push lawrencegripper/ion-module-downloader 
+
+```
+
+Now we can use the ION cli to create a module in our pipeline from this docker image. The `-i` and `-o` arguments represent the `input` and `output` events. 
+The `frontapi.new_link` event in outputted by the [FrontAPI used to submit items to the pipeline](./../cmd/frontapi). The `file_downloaded` event is what we'll output
```
Comment (jjcollinge) [inline docs/Quick Start.md]: in => is

File: docs/Quick Start.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+after we've downloaded the file. Lastly the `-p` specifies the provider to run the module under. In this case we'll use a shared Kuberentes environment and it will run as a [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/).
```
Comment (jjcollinge) [inline docs/Quick Start.md]: "Lastly the `-p` flag specifies..."

File: docs/Quick Start.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+``` bash 
+
+ion module create --module-image lawrencegripper/ion-module-transcode -i file_downloaded -o file_transcoded -p AzureBatch
+
+```
+
+# 3. Submit a link to the pipeline 
+
+
+So next lets trigger our pipeline with a new URL to download and transcode. We'll use CURL to POST [out request](./data/examplesubmission.json) for the [Big Buck Bunny film](https://peach.blender.org/).
```
Comment (jjcollinge) [inline docs/Quick Start.md]: out => our

File: docs/Quick Start.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+```
+
+> *NOTE*: Update the command and put your `correlationId` in place of `your-correlation-id-here` for example `0bff834f-4dfc-484a-80ed-4f2cfc149c23` from the response above. 
+
+Congratulations, you've run a pipeline in ION. You'll now get information back about the execution of the pipeline. 
+
+Here is a quick guide to reading this output. 
+
+- `context.documentType` can be either:
+    - `eventMeta`: This is data transferred between events used by the system. Then include links to blob data uploaded by a module so you can debug the flow between modules.
```
Comment (jjcollinge) [inline docs/Quick Start.md]: Then include => This includes

File: docs/Quick Start.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+> *NOTE*: Update the command and put your `correlationId` in place of `your-correlation-id-here` for example `0bff834f-4dfc-484a-80ed-4f2cfc149c23` from the response above. 
+
+Congratulations, you've run a pipeline in ION. You'll now get information back about the execution of the pipeline. 
+
+Here is a quick guide to reading this output. 
+
+- `context.documentType` can be either:
+    - `eventMeta`: This is data transferred between events used by the system. Then include links to blob data uploaded by a module so you can debug the flow between modules. 
+    - `insight`: This is data stored by the module for querying by the user, this could include execution time, items spotted in the video or any arbitrary data for query. You can inspect the putput here.
```
Comment (jjcollinge) [inline docs/Quick Start.md]: putput => output

**Review by jjcollinge — CHANGES_REQUESTED** [review]
A few typos. Should we link to this from the main README.md too? All good otherwise

**Comment by lawrencegripper** [comment]
Thanks, yeah good idea I've added in a link in the root readme and hopefully fixed up those typos.

**Review by jjcollinge — APPROVED** [review]

**Pull request merged**